draft-kwbdgrr-tsvwg-net-collab-rqmts.md

```diff
@@ -299,8 +299,7 @@ Intentional Management:
 quota or bandwidth limit, or quality (delay and/or jitter)) assurances.
 
 Reactive Management:
-: Network reactions to congestion events or
-protection polices under attacks, with very short to very long
+: network policies which react to congestion events with very short to very long
```

Comment:
```suggestion
: Network management actions that are undertaken as a reaction to unplanned overload events. Concretely, this includes policies which react to congestion events with very short to very long durations (e.g., varying wireless and mobile air interface conditions) or protection policies to soften the impact of ongoing attacks.
```

Reply:
Dan's def is restrictive, IMO hence the proposed revised text.

Reply:
Med's revision is clear.